common/saber_base.h

```diff
@@ -68,7 +69,11 @@ class SaberBase {
   // 1.0 = kDefaultVolume
   // This is really just for sound fonts.
   virtual void SetHumVolume(float volume) {}
-
+  virtual void StartSwing(Effect* monophonic, Effect* polyphonic) {}
```

Comment:
This function does not need to send in effect pointers.
HybridFont already knows what effects to use to start a swing.

Reply:
fixed.